### PR TITLE
display file name not full path in balloon notification

### PR DESCRIPTION
**Pull request by Atlas139mkm**
### Description of the Change

@GammaVel  hopefully this looks a bit better.

As per #1309 notification being to big due to the file path I have removed the path. Now it only displays the file names separated by a comma.

image
(it says the same file 3 times as its the only one I had. I added it 3 times)

### Alternate Designs

Have all file names on separate lines, however this was not possible due to the notification styles restrictions.

### Why Should This Be In Core?

Easier to read.

### Benefits

UX UI

### Possible Drawbacks

Can't see the full path in the notification. However they are still present in the file importer view.

### Verification Process

Ensured all relevant file names were posted

### Applicable Issues

#1309

### CODE REVIEW

**Review by arcturus2 — APPROVED** [review]

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java [hunk 1]
```diff
@@ -147,28 +147,38 @@ private void displaySummaryAlert(final int importedRows, final List<String> vali
...
-                    sbMessage.append("\n  ");
-                    sbMessage.append(filename);
-                });
-                final String invalidFileFiles = (validFilenames.size() == 1) ? "file" : "files";
+                sbMessage.append("The following " + fileFiles + " contained data: ");
+
+                for (int i = 0; i < validFilenames.size(); i++) {
+                    final File f = new File(validFilenames.get(i));
+                    sbMessage.append(f.getName());
+                    if (validFilenames.size() - 1 != i) {
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java:155]: are you able to move `i` to the LHS of this boolean statement?

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java [hunk 2]
```diff
@@ -147,28 +147,38 @@ private void displaySummaryAlert(final int importedRows, final List<String> vali
...
-                    sbMessage.append("\n\nThe following " + invalidFileFiles + " could not be parsed. No data was extracted:");
-                    invalidFilenames.forEach(filename -> {
-                        sbMessage.append("\n  ");
-                        sbMessage.append(filename);
-                    });
+                    sbMessage.append("\n\nThe following " + invalidFileFiles + " could not be parsed. No data was extracted: ");
+                    for (int i = 0; i < invalidFilenames.size(); i++) {
+                        final File f = new File(invalidFilenames.get(i));
+                        sbMessage.append(f.getName());
+                        if (validFilenames.size() - 1 != i) {
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java:166]: similar here

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java [hunk 3]
```diff
@@ -147,28 +147,38 @@ private void displaySummaryAlert(final int importedRows, final List<String> vali
...
-                sbMessage.append("The following " + fileFiles + " could not be parsed. no data was extracted:");
-                invalidFilenames.forEach(filename -> {
-                    sbMessage.append("\n  ");
-                    sbMessage.append(filename);
-                });
+                sbMessage.append("The following " + fileFiles + " could not be parsed. no data was extracted: ");
+                for (int i = 0; i < invalidFilenames.size(); i++) {
+                    final File f = new File(invalidFilenames.get(i));
+                    sbMessage.append(f.getName());
+                    if (validFilenames.size() - 1 != i) {
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java:178]: and here

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by antares1470 — APPROVED** [review]
I didn't even pick up the fact that the last 2 should've been comparing to `invalidFilenames` rather than `validFilenames` so good pick up

**Pull request merged**